cobalt/cobalt.cc

```diff
@@ -39,15 +39,15 @@ int main(int argc, const char** argv) {
   // parameters, also update the my_argc below.
   static const char* my_argv[] = {
       argv[0], "--disable-fre", "--no-first-run", "--kiosk",
-      "--force-video-overlays",
+      "--force-video-overlays", "--single-process",
       // Enable remote devtools access.
       "--remote-debugging-port=9222",
       "--remote-allow-origins=http://localhost:9222",
       // This flag is added specifically for m114 and should be removed after
       // rebasing to m120+
       "--user-level-memory-pressure-signal-params",
       "https://www.youtube.com/tv", nullptr, nullptr};
-  int my_argc = 9;
+  int my_argc = 10;
```

Comment:
Can we use `std::array<char*>(my_argv).count() -3` or so?

Reply:
Yes there are many ways to improve this, See b/375241103 mentioned above. This way it's super obvious that this is only a temporary approach. This CL does not change that, just adds a parameter.